services/actions/ReserveIps.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
why are these action hooks added in bosh release? They should just reside in broker code or product component.

Reply:
If the intention was to have a sample for external contributors, then may be 'render' function which generates the manifest should also be modified ?

Reply:
Done

Reply:
ok